saleor/plugins/webhook/tests/test_webhook.py

```diff
@@ -41,7 +41,7 @@ def test_trigger_webhooks_for_event(
     }
 
     mock_request.assert_called_once_with(
-        webhook.target_url, data=expected_data, headers=expected_headers, timeout=10
+        webhook.target_url, json=expected_data, headers=expected_headers, timeout=10
```

Comment:
Maybe I don't remember but I don't understand why we are mocking a request if we have enabled `vcr`

Reply:
Well if I have to guess it is just another check for arguments for request POST

Reply:
Shouldn't the tests fail if the VCR are outdated?

Reply:
@NyanKiyoshi yes and it fails as you can see above.

Reply:
Ach, yes! I've just saw the `wraps`

Reply:
@NyanKiyoshi wait what... on test passed while on local it was failing. I have just recreated VCRs...
The second commit should fail.